src/aes/encrypt.ts

```diff
@@ -1,30 +1,20 @@
 import crypto from 'crypto';
 
-interface AESInit {
-  iv: string;
-  salt: string;
-}
-
 /**
  *
  * @param text Text to encrypt
  * @param password
- * @param aesInit Object that contains IV + Salt. If its null, random bytes will be used.
- * @param hops
  * @returns Encrypted text
  */
-export default function encrypt(text: string, password: string, aesInit: AESInit | null = null, hops = 2145): string {
-  // random initialization vector
-  const iv = aesInit ? Buffer.from(aesInit.iv || '', 'hex') : crypto.randomBytes(16);
-
+export default function encrypt(text: string, password: string): string {
+ 
   // random salt
-  const salt = aesInit ? Buffer.from(aesInit.salt || '', 'hex') : crypto.randomBytes(64);
+  const salt = crypto.randomBytes(64);
```

Comment:
@sg-gs It doesn't have to be 64 bytes. Usually, people use up to 32. I think the current recommendation by NIST for a similar construction is 16 bytes.